### ISSUE TITLE
Critical security vulnerability in vm2

### ISSUE THREAD

**Issue by Havunen** [TooTallNate/node-pac-resolver#28]
Sandbox bypass in vm2 - https://github.com/advisories/GHSA-6pw2-5hjv-9pf7
fix available via `npm audit fix`

**node_modules/vm2**

1 critical severity vulnerability

 **npm list vm2** shows this repository as part of the dependency chain

```

`-- ibm-openapi-validator@0.53.1
  `-- @stoplight/spectral-cli@6.2.0
    `-- proxy-agent@5.0.0
      `-- pac-proxy-agent@5.0.0
        `-- pac-resolver@5.0.0
          `-- degenerator@3.0.1
            `-- vm2@3.9.5

```

Updating degenerator should fix the problem see: https://github.com/TooTallNate/node-degenerator/releases/tag/3.0.2

**Comment by Havunen** [comment]
Opened PR to fix this issue: https://github.com/TooTallNate/node-pac-resolver/pull/29

**Comment by dgilperez** [comment]
Same here, it would be great if this can go forward 🙏 😄

**Comment by toastyghost** [comment]
+1 thanks for the update on degenerator but we're using this as a deep subdep of winston-cloudwatch so the whole chain needs to be updated: winston-cloudwatch > proxy-agent > pac-proxy-agent > pac-resolver > degenerator (done) > vm2 (done)

**Comment by Havunen** [comment]
@TooTallNate Could you review the change and publish new version please?

**Comment by RubyKozel** [comment]
What's the status of this?

**Comment by Havunen** [comment]
Maintainer of this project is not responding

**Comment by Havunen** [comment]
@TooTallNate

**Comment by Havunen** [comment]
fixed in 5.0.1
